.github/workflows/ci.yml

```diff
@@ -43,12 +43,8 @@ jobs:
           # Don't use a 'components:' entry--we don't need them with beta/nightly, plus nightly often doesn't have them
           override: true
 
-      # This block can be uncommented to add clippy to CI, but first the existing clippy warnings need to be fixed!
-      # - name: clippy
-      #   run: |
-      #     rustup component add clippy
```

Comment:
do we no longer need to add clippy anymore?

Reply:
I'm not sure I understand. This pull request adds the `cargo clippy` check back by un-commenting this code out.